pkg/upstream/cluster/cluster_manager.go

```diff
@@ -173,6 +201,7 @@ func (cm *clusterManager) AddOrUpdateClusterAndHost(cluster v2.Cluster, hostConf
 		UpdateClusterResourceManagerHandler(oc, nc)
 		CleanOldClusterHandler(oc, nc)
 		NewSimpleHostHandler(nc, hostConfigs)
+		TransferClusterHostStatesHandler(oc, nc)
```

Comment:
trans it only when slow start is enabled. Same elsewhere.

Reply:
I will deal with this later.

Reply:
Done